src/test/java/net/snowflake/client/log/JDK14LoggerLatestIT.java

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
This file is exactly the same as JDK14LoggerIT.java except the single test I added.